recipes/qtkeychain/all/conanfile.py

```diff
@@ -29,20 +31,33 @@ class PackageConan(ConanFile):
     def layout(self):
         cmake_layout(self, src_folder="src")
 
+    def validate(self):
+        check_min_cppstd(self, 17)
+        qt_options = self.dependencies["qt"].options
+        if self.settings.os in ("Linux", "FreeBSD") and not qt_options.with_dbus:
```

Comment:
Im having some problems testing this configuration currently, will dig deeper on Monday